web/marketing/pages/freeOffers.html

```diff
@@ -1,51 +1,58 @@
 {{define "freeOffers"}}
-<table class="w-100 mt-5 mb-5 rounded table table-light table-responsive">
-        <tr>
-            <th>Name</th>
-            <th>Give Credit</th> 
-            <th>Free Credits Used</th>
-            <th>Redeemable Capacity</th>
-            <th>Created</th>
-            <th>Expiration</th>
-            <th>Status</th>
-        </tr>
-        <th class="text-dark m-2"><p>DEFAULT&nbsp;OFFER</p></th>
-        <tr>
-            <td>Default Free Credit</td>
-            <td>$20</td> 
-            <td>48</td>
-            <td>N/A</td>
-            <td>04/18/19</td>
-            <td>N/A</td>
-            <td><img class="toggler" src="/static/img/on.jpg" height="50" width="50"></td>
-        </tr>
-        <th class="text-dark m-2"><p>CUSTOM&nbsp;OFFER(S)</p></th>
-        <tr>
-            <td>May Free Credit</td>
-            <td>$50</td> 
-            <td>30</td>
-            <td>200</td>
-            <td>05/18/19</td>
-            <td>06/18/2019</td>
-            <td><img class="toggler" src="/static/img/off.jpg" height="50" width="50"></td>
-        </tr>
-        <tr>
-            <td>June Free Credit</td>
-            <td>$40</td> 
-            <td>0</td>
-            <td>200</td>
-            <td>05/18/19</td>
-            <td>06/18/2019</td>
-            <td><img class="toggler" src="/static/img/off.jpg" height="50" width="50"></td>
-        </tr>
-        <tr>
-            <td>July Free Credit</td>
-            <td>$30</td> 
-            <td>0</td>
-            <td>200</td>
-            <td>07/01/19</td>
-            <td>08/01/2019</td>
-            <td><img class="toggler" src="/static/img/off.jpg" height="50" width="50"></td>
-        </tr>
-    </table>
+<div class="offers-table mt-2 mb-5 container">
+        <div class="row">
+            <div class="col col-heading">Name</div>
+            <div class="col col-heading">Give Credit</div> 
+            <div class="col col-heading">Get Credit</div>
+            <div class="col col-heading">Referrals&nbsp;Used</div>
+            <div class="col col-heading">Redeemable&nbsp;Capacity</div>
+            <div class="col col-heading">Created</div>
+            <div class="col col-heading">Expiration</div>
+            <div class="col col-heading">Status</div>
+            <div class="col col-heading edit-offer"></div>
+        </div>
+        <hr>
+        <div class="row offer-heading ">
+            <p class="offer-type">DEFAULT&nbsp;OFFER</p>
+        </div>
+        <div class="row data-row">
+            <div class="col ml-3">May Referral</div>
```

Comment:
Is the dynamic template going to be built in another PR? Right now all the data is hard-coded

Reply:
Yes I already have all that done. It will be included in the Create-offer PR. I just wanted to keep the code separate to have smaller PR's.